docs/Makefile

```diff
@@ -85,17 +85,17 @@ qthelp:
 	@echo
 	@echo "Build finished; now you can run "qcollectiongenerator" with the" \
 	      ".qhcp project file in $(BUILDDIR)/qthelp, like this:"
-	@echo "# qcollectiongenerator $(BUILDDIR)/qthelp/pyrlp.qhcp"
+	@echo "# qcollectiongenerator $(BUILDDIR)/qthelp/rlp.qhcp"
```

Comment:
Curious about this change since I think it probably started as a template var but now matches the package name? Assuming the package name didnt change, was this working before?

Reply:
fixed